packages/babel-preset-env/src/index.js

```diff
@@ -115,114 +216,65 @@ export default declare((api, opts) => {
 
   const transformTargets = forceAllTransforms || hasUglifyTarget ? {} : targets;
 
-  const transformations = filterItems(
+  const modulesPluginNames = getModulesPluginNames({
+    moduleTransformation: moduleTransformations[modules.toString()] || null,
```

Comment:
nit: I am upset by `false.toString()` can be passed into `moduleTransformations`, though practically it does not change anything.

I prefer to pass `moduleTransformation` and `modules` instead and do the logic inside `getModulesPluginNames`.

Reply:
Not ideal I know! This is one of those situations where using flow makes things less elegant (my own opinion ™). 

Made the change you proposed in my latest commit. Used the variable name `transformations` instead of `moduleTransformations` inside of `getModulesPluginNames` so that it not gets confused with the `moduleTransformations` variable in the global scope.